test/integration/test-amp-ad-fake.js

```diff
@@ -32,7 +32,7 @@ describe('A4A', function() {
       `,
     extensions: ['amp-ad'],
   }, () => {
-    it('should layout amp-img, amp-pixel, amp-analytics', () => {
+    it.skip('should layout amp-img, amp-pixel, amp-analytics', () => {
```

Comment:
The test that's flaking is not this one, but the one at https://github.com/ampproject/amphtml/blob/master/test/integration/test-amp-inabox.js#L164.

Though I'm not sure why that flakes specifically; all the other BTF tests with more exotic setups doesn't.

Reply:
😱 thanks for finding this! >__< embarrassing of me to disable the wrong flaky test...